product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -27,6 +27,27 @@ To prevent failures while creating your clusters, ensure that each of the follow
 | Microsoft.OperationsManagement | Monitors workloads and provides container insight                                                      |
 | Microsoft.Portal               | Provides a dashboard to monitor the running status of the clusters (using aggregated logs and metrics) |
 
+## Regional activation resources and access requirements
+
+The table provides BigAnimal's regional activation resources and access requirements.
+
+| Resource type                        | Activated region | Network access | Description                                                          |
+|--------------------------------------|------------------|----------------|----------------------------------------------------------------------|
+| Virtual network                      | Yes              | N/A            |                                                                      |
+| Storage account for backup           | yes              | Public         |                                                                      |
+| KeyVault for backup, AKS             | Yes              | Public         |                                                                      |
+| Storage account for logs and metrics | Yes              | Private        | Public is required for faraway replication and cross region restore. |
```

Comment:
"Public is required for faraway replication and cross region restore." should be moved to the Description of 37 - Storage account for backup